src/pybind/mgr/telemetry/tests/telemetry_upgrade_tests.md

```diff
@@ -0,0 +1,213 @@
+The `upgrade` suite is used to verify that upgrades from the (X-2) releases of
+Ceph (where X stands for the current release, i.e tentacle as of writing this
+documentation) to the current stable Ceph release (_tentacle_) can complete
+successfully without disrupting any ongoing workloads.
```

Comment:
```suggestion
The `upgrade` suite is used to verify that upgrades can complete
successfully without disrupting any ongoing workloads.
```

Let's simplify this introduction since you explain the X-2 aspect a bit later in the doc.